frontend/hub/namespaces/HubNamespaceForm.tsx

```diff
@@ -48,19 +51,22 @@ export function EditHubNamespace() {
   const navigate = useNavigate();
   const params = useParams<{ id?: string }>();
   const name = params.id;
-  const { data: namespace } = useGet<HubNamespace>(hubAPI`/_ui/v1/namespaces/${name ?? ''}/`);
```

Comment:
If there is an error when fetching the error, and during the loading we could.

```
  const
{     data: namespace,     error,     refresh,   }

= useGet<ItemsResponse<HubNamespace>>(
    pulpAPI`/pulp_ansible/namespaces/?name=${name ?? ''}`
  );

  if (error) return <AwxError error={error} handleRefresh={refresh} />;
  if (!task) return <LoadingPage breadcrumbs tabs />;

```

I have seen this pattern across the code basis.